app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> None:
+    sep_phrases = command.split()
```

Comment:
```suggestion
    parts = command.split()
```
Rename it, because sep_phrases is not clear name